src/frontend/sharedComponents/ProjectInviteBottomSheet/InviteCanceledBottomSheetContent.tsx

```diff
@@ -0,0 +1,52 @@
+import * as React from 'react';
+import {BottomSheetContent} from '../BottomSheetModal';
+import {defineMessages, useIntl} from 'react-intl';
+import Error from '../../images/Error.svg';
+
+const m = defineMessages({
+  close: {
+    id: 'sharedComponents.ProjectInviteBottomSheet.InviteCancelled.close',
+    defaultMessage: 'Close',
+  },
+  canceled: {
+    id: 'sharedComponents.ProjectInviteBottomSheet.InviteCancelled.canceled',
+    defaultMessage: 'Invite Canceled',
+  },
+  projectInviteCanceled: {
+    id: 'sharedComponents.ProjectInviteBottomSheet.InviteCancelled.projectInviteCanceled',
+    defaultMessage: 'Your invitation to {projName} has been canceled.',
```

Comment:
nit: no need to be terse with the placeholder name. would instead use `projectName`

Reply:
https://github.com/digidem/comapeo-mobile/pull/366/commits/a5e25113e5b74d9b925c9b6f8fc79639206a0e4e